### PR TITLE
Adaptation of the truth table according to #873

### PR DESCRIPTION
**Pull request by mbaillif**
Adaptation of the truth table according to #873

Addition of the FR version
HTML normalization for RU/PT

* #873

### CODE REVIEW

**Comment by mbaillif** [comment]
image

**Comment by mbaillif** [comment]
After : 
image

**Review by BFH-ktt1 — DISMISSED** [review]

**Comment by BFH-ktt1** [comment]
DEPENDS_ON #903

**Comment by MarcinOrlowski** [comment]
`DEPENDS_ON` and `BLOCKED_BY` works only when put in PR description. Commit messages or additional comments are not checked. I edited PR description.

**Comment by mbaillif** [comment]
Thank you

File: src/main/resources/doc/en/html/libs/gates/controlled.html [hunk 1]
```diff
@@ -74,86 +74,190 @@ <h2>
...
       </p>
+	  <div class="attliste">
       <dl>
-        <dt>
+        <dt class="lib">
           Facing
         </dt>
         <dd>
-          The direction of the component (its output relative to its input).
+          The direction of the component, its output relative to its input.
```
Comment (maehne) [inline src/main/resources/doc/en/html/libs/gates/controlled.html:227]: I think the original description with parentheses is clearer. @davidhutchens: What do you think as a native speaker?

File: src/main/resources/doc/fr/html/libs/gates/controlled.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+              </table>
+			  </td>
+          </tr>
+        </tbody>
+      </table>
+      <h2>
+        Comportement
+      </h2>
+      <p>
+		Le tampon et l'inverseur contrôlés, souvent appelés <em>tampon/inverseurs à trois états</em>, possèdent chacun une broche d'entrée "Contrôle" d'un bit sur le coté. La valeur sur cette broche de commande affecte le comportement du composant :
```
Comment (maehne) [inline src/main/resources/doc/fr/html/libs/gates/controlled.html:77]: Consider using a non-breaking space before colon at the end of this line.

File: src/main/resources/doc/fr/html/libs/gates/controlled.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+          Orientation</b>
+        </dt>
+        <dd>
+          La direction du composant, sa sortie par rapport à son entrée.
+        </dd>
+        <dt class="lib">
+          largeur de données</b>
+        </dt>
+        <dd>
+          La largeur de bit de l'entrées et la sorties du composant.
```
Comment (maehne) [inline src/main/resources/doc/fr/html/libs/gates/controlled.html:233]: Shouldn't it be singular for input and output, i.e., "La largeur de bit de l'entrée et de la sortie du composant."?

File: src/main/resources/doc/fr/html/libs/gates/controlled.html [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+          Dimension dessins</b>
+        </dt>
+        <dd>
+          (Tampon controlé inverseur seulement) Détermine s'il faut dessiner une version plus grande ou plus petite du composant.
+        </dd>
+        <dt class="lib">
+          Position de la commande
+        </dt>
+        <dd>
+   		   L'emplacement de la ligne de contrôle, en imaginant que nous regardons la sortie depuis l'entrée : Si le composant est orienté vers l'est et est règlé <q>A droite,</q> la ligne de contrôle est au sud ; mais s'il est règlé <q>A gauche,</q> la ligne de contrôle est au nord.
```
Comment (maehne) [inline src/main/resources/doc/fr/html/libs/gates/controlled.html:245]: Shouldn't it be "À droite" et "À gauche"?

**Review by maehne — CHANGES_REQUESTED** [review]
Looks good to me except for some small issues I pointed out in the comments. Though, I am not able to comment on the modifications, which you did to the Portuguese and Russian localisation. Thanks @mbaillif for your efforts!

File: src/main/resources/doc/fr/html/libs/gates/controlled.html (hunk 4 above)
Comment (mbaillif) [reply src/main/resources/doc/fr/html/libs/gates/controlled.html]: Indeed, the new French spelling requires it.
I correct it

**Comment by BFH-ktt1** [comment]
Dear all, this PR is still blocked due to an open unresolved conversation. Only in case it is resolved I can merge.

**Comment by MarcinOrlowski** [comment]
@mbaillif can you please either apply @maehne requested changes, reject them or whatever you like and close related conversations. Because for now it's unknown what is your take on these.

File: src/main/resources/doc/en/html/libs/gates/controlled.html (hunk 1 above)
Comment (mbaillif) [reply src/main/resources/doc/en/html/libs/gates/controlled.html]: ok

File: src/main/resources/doc/en/html/libs/gates/controlled.html [hunk 5]
```diff
@@ -224,7 +224,7 @@ <h2>
           Facing
         </dt>
         <dd>
-          The direction of the component, its output relative to its input.
+          The direction of the component, its output relative to it input.
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/en/html/libs/gates/controlled.html:227]: That's not correct, so please roll back. And after a thought, please leave "its" as it was before. Sorry for the mess.

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/resources/doc/en/html/libs/gates/controlled.html (hunk 5 above)
Comment (mbaillif) [reply src/main/resources/doc/en/html/libs/gates/controlled.html:227]: The fault is on my side, I used directly GitHub web and it does not suit me.

**Review by MarcinOrlowski — DISMISSED** [review]

**Review by mbaillif — COMMENTED** [review]
A mistake :-(

File: src/main/resources/doc/en/html/libs/gates/controlled.html [hunk 6]
```diff
@@ -57,10 +57,10 @@ <h1>
                 <tbody>
 				  <tr>
 				     <td>
-                      <img class="appearancelibs" src="../../../../img-libs/anscontrolledBuffer.png" alt="#########" height="64" width="64">
+                      <img class="appearancelibs" src="../../../../img-libs/anscontrolledbuffer.png" alt="#########" height="64" width="64">
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/en/html/libs/gates/controlled.html:60]: Please remove `alt="#########"`(multiple places)

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/resources/doc/en/html/libs/gates/controlled.html (hunk 6 above)
Comment (maehne) [reply src/main/resources/doc/en/html/libs/gates/controlled.html:60]: ... or replace by a descriptive text als alternative representation of the image.

File: src/main/resources/doc/en/html/guide/prefs/pref-intl.html [hunk 7]
```diff
@@ -61,7 +61,7 @@ <h1>
             </table>
           </center>
           <p>
-            Because the <i>IEC</i> style tends to be more popular in the U.S., while the <i>ANSI<(i>) style tends to be more popular in Europe, some people refer to these styles according to these regions. <!-- The DIN 40700 standard was a standard for drafting digital and analog electronic components adopted by DIN, a German standards organization. DIN adopted the rectangular standard for digital components in 1976, but some engineers continue to use the older style; they appear to be increasingly rare.-->
+            Because the <i>IEC</i> style tends to be more popular in the U.S., while the <i>ANSI</i> style tends to be more popular in Europe, some people refer to these styles according to these regions. <!-- The DIN 40700 standard was a standard for drafting digital and analog electronic components adopted by DIN, a German standards organization. DIN adopted the rectangular standard for digital components in 1976, but some engineers continue to use the older style; they appear to be increasingly rare.-->
```
Comment (maehne) [inline src/main/resources/doc/en/html/guide/prefs/pref-intl.html:64]: IEC and ANSI are mixed up here! It should be:

"Because the <i>ANSI</i> style tends to be more popular in the U.S., while the <i>IEC</i> style tends to be more popular in Europe, some people refer to these styles according to these regions."

Please check whether other parts/localisations are affected, too!

File: src/main/resources/doc/en/html/libs/gates/controlled.html [hunk 8]
```diff
@@ -74,86 +74,190 @@ <h2>
...
+          The component's control input. Bit width 1.
         </dd>
-        <dt>
-          East edge (output, bit width matches Data Bits attribute)
+        <dt class="lib">
+          East edge:
         </dt>
         <dd>
-          The component's output, which will be floating if the control input is 0 or floating, the error value if the control input is the error value, and will be computed based on the west-side input if the control input is 1.
+          The component's output, which will be floating (<b class="uvalue">U</b>) if the control input is <b class="zerov">0</b> or floating (<b class="uvalue">U</b>), the error value (<b class="evalue">E</b>) if the control input is the error value (<b class="evalue">E</b>), and will be computed based on the west-side input if the control input is <b class="unov">1</b>.  Bit width matches  <b class="propertie">Data Bits</b> attribute.
```
Comment (maehne) [inline src/main/resources/doc/en/html/libs/gates/controlled.html:211]: "Bit width matches  <b class="propertie">Data Bits</b> attribute." -> "The bit width matches  <b class="propertie">Data Bits</b> attribute."

PS: There's a typo in the class name: "propertie" -> "property". Maybe this would need fixing in more files. Then, this should be done in a separate PR.

File: src/main/resources/doc/fr/html/guide/prefs/pref-intl.html [hunk 9]
```diff
@@ -61,7 +61,7 @@ <h1>
             </table>
           </center>
           <p>
-            Selon les continents un style à la préférence. "IEC" est plus populaire aux Etats-Unis et ANSI est préféré en Europe. Certaines personnes se réfèrent à ces styles selon ces régions.
+            Selon les continents un style à la préférence. <i>IEC</i> est plus populaire aux Etats-Unis et <i>ANSI</i> est préféré en Europe. Certaines personnes se réfèrent à ces styles selon ces régions.
```
Comment (maehne) [inline src/main/resources/doc/fr/html/guide/prefs/pref-intl.html:64]: See above: "IEC" and "ANSI" are mixed up!

**Review by maehne — CHANGES_REQUESTED** [review]
The modifications to the English and French documentation look sensible to me besides the mix up of "IEC" and "ANSI". I cannot comment on the other touched languages. Please, fix issues simular to the ones I noted in the English and French translations there, too. 

@mbaillif: Thanks for your continuous work on improving the documentation!

File: src/main/resources/doc/en/html/guide/prefs/pref-intl.html [hunk 10]
```diff
@@ -25,7 +25,7 @@ <h1>
       <ul>
         <li>
           <p>
-            <b>Gate shape:</b> Logisim supports two standards for drawing gates: <i>IEC</i> and <i>ANSI</i>. The following table illustrates the distinction.
+            <b>Gate shape:</b> Logisim supports two standards for drawing gates: <i>IEC</i> gates, <i>ANSI</i>. The following table illustrates the distinction.
```
[the remark this answers is not in the document]
Comment (mbaillif) [reply src/main/resources/doc/en/html/guide/prefs/pref-intl.html]: Thank you for this proofreading.
This point is also announced in #962

File: src/main/resources/doc/en/html/guide/prefs/pref-intl.html (hunk 7 above)
Comment (mbaillif) [reply src/main/resources/doc/en/html/guide/prefs/pref-intl.html]: As before

File: src/main/resources/doc/fr/html/guide/prefs/pref-intl.html (hunk 9 above)
Comment (mbaillif) [reply src/main/resources/doc/fr/html/guide/prefs/pref-intl.html]: As before

File: src/main/resources/doc/en/html/libs/gates/controlled.html (hunk 8 above)
Comment (mbaillif) [reply src/main/resources/doc/en/html/libs/gates/controlled.html:211]: Indeed, it's a style name that I messed up a bit.
I will correct it but it will impact (195) many files.
But it works.

**Comment by MarcinOrlowski** [comment]
What is the status of this one?

**Review by BFH-ktt1 — APPROVED** [review]

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski , @maehne : Can you unblock please?

**Comment by MarcinOrlowski** [comment]
Changes are blocking but it's PR of @mbaillif

**Review by MarcinOrlowski — APPROVED** [review]

**Comment by MarcinOrlowski** [comment]
@mbaillif what's the fate of this PR?

**Review by maehne — APPROVED** [review]
Thanks @mbaillif! Looks good to me now. Sorry for the very long delay it took me to review this PR again!
